pkg/image/remote/remote.go

```diff
@@ -113,13 +119,16 @@ func RetrieveRemoteImage(image string, opts config.RegistryOptions, customPlatfo
 	return remoteImage, err
 }
 
-// normalizeReference adds the library/ prefix to images without it.
+// normalizeReference adds the library/ or the {path}/ in registry map suffix to images without it.
 //
-// It is mostly useful when using a registry mirror that is not able to perform
-// this fix automatically.
-func normalizeReference(ref name.Reference, image string) (name.Reference, error) {
+// It is mostly useful when using a registry maps that is not able to perform
+// this fix automatically add library or the custom path on registry map.
+func normalizeReference(ref name.Reference, image string, custompath string) (name.Reference, error) {
+	if custompath == "" {
+		custompath = "libary"
```

Comment:
Sorry, I am not a maintainer, but I noticed a typo here: a `r` is missing in `library`

Reply:
ahh yeah thankyou